tests/integration/_fixtures/upload_debugid_alias/server/chunks/1.js.map

```diff
@@ -0,0 +1 @@
le.prototype.__init2.call(this);\n    if (options.levels) {\n      this._levels = options.levels;\n    }\n  }\n\n  /**\n   * @inheritDoc\n   */\n   setupOnce(_, getCurrentHub) {\n    if (!('console' in utils.GLOBAL_OBJ)) {\n      return;\n    }\n\n    this._levels.forEach((level) => {\n      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any\n      if (!(level in (utils.GLOBAL_OBJ ).console)) {\n        return;\n      }\n\n      // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access\n      utils.fill((utils.GLOBAL_OBJ ).console, level, (originalConsoleMethod) => (...args) => {\n        const hub = getCurrentHub();\n\n        if (hub.getIntegration(CaptureConsole)) {\n          hub.withScope(scope => {\n            scope.setLevel(utils.severityLevelFromString(level));\n            scope.setExtra('arguments', args);\n            scope.addEventProcessor(event => {\n              event.logger = 'console';\n              return event;\n            });\n\n            let message = utils.safeJoin(args, ' ');\n            if (level === 'assert') {\n              if (args[0] === false) {\n                message = `Assertion failed: ${utils.safeJoin(args.slice(1), ' ') || 'console.assert'}`;\n                scope.setExtra('arguments', args.slice(1));\n                hub.captureMessage(message);\n              }\n            } else if (level === 'error' && args[0] instanceof Error) {\n              hub.captureException(args[0]);\n            } else {\n              hub.captureMessage(message);\n            }\n          });\n        }\n\n        // this fails for some browsers. :(\n        if (originalConsoleMethod) {\n          originalConsoleMethod.apply(utils.GLOBAL_OBJ.console, args);\n        }\n      });\n    });\n  }\n} CaptureConsole.__initStatic();\n\nexports.CaptureConsole = CaptureConsole;\n//# sourceMappingURL=captureconsole.js.map\n","Object.defineProperty(exports, '__esModule', { value: true });\n\nconst utils = require('@sentry/utils');\n\n/**\n * Integration to debug sent Sentry events.\n * This integration should not be used in production\n */\nclass Debug  {\n  /**\n   * @inheritDoc\n   */\n   static __initStatic() {this.id = 'Debug';}\n\n  /**\n   * @inheritDoc\n   */\n   __init() {this.name = Debug.id;}\n\n   constructor(options) {Debug.prototype.__init.call(this);\n    this._options = {\n      debugger: false,\n      stringify: false,\n      ...options,\n    };\n  }\n\n  /**\n   * @inheritDoc\n   */\n   setupOnce(addGlobalEventProcessor, getCurrentHub) {\n    addGlobalEventProcessor((event, hint) => {\n      const self = getCurrentHub().getIntegration(Debug);\n      if (self) {\n        if (self._options.debugger) {\n          // eslint-disable-next-line no-debugger\n          debugger;\n        }\n\n        /* eslint-disable no-console */\n        utils.consoleSandbox(() => {\n          if (self._options.stringify) {\n         
```

Comment:
The test hinges on the fact that this sourcemap has contains a debug ID under `"debugId"`.